src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceArr = sourceString
+    .split(';')
+    .map((element) => element.trim())
+    .filter((element) => element !== '');
+
+  const diferentArr = sourceArr.map((style) => style.split(':'));
```

Comment:
The variable name `diferentArr` is not very descriptive and also contains a typo. Consider using a more meaningful and correctly spelled name to improve code readability, as per the checklist item: 'Avoid variable namings like `array`, `object`, `arr`, `obj`, `result`.'